dev/stage-binaries.sh

```diff
@@ -17,10 +17,16 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
+SCALA_VERSION=2.12
 FLINK_VERSIONS=1.12,1.13,1.14
 SPARK_VERSIONS=2.4,3.0,3.1,3.2
 HIVE_VERSIONS=2,3
 
-./gradlew -Prelease -DflinkVersions=$FLINK_VERSIONS -DsparkVersions=$SPARK_VERSIONS -DhiveVersions=$HIVE_VERSIONS publishApachePublicationToMavenRepository
+./gradlew -Prelease -DscalaVersion=$SCALA_VERSION -DflinkVersions=$FLINK_VERSIONS -DsparkVersions=$SPARK_VERSIONS -DhiveVersions=$HIVE_VERSIONS publishApachePublicationToMavenRepository
+
+# Also publish Scala 2.13 Artifacts for versions that support it.
+# Flink does not yet support 2.13 and Hive doesn't need a Scala specification.
+SCALA_VERSION=2.13
+SPARK_VERSIONS_2.13=3.2
```

Comment:
Adding `_2.13` is a bit confusing since the Scala version is parameterized. What about just redefining `SPARK_VERSIONS` like you do with `SCALA_VERSION`?

Reply:
Removing these anyway for now, but good call.